gitlab/pkg/commands/helpers.go

```diff
@@ -0,0 +1,11 @@
+package commands
+
+import "strings"
+
+func IndentString(s string) string {
+	lines := strings.Split(s, "\n")
+	for i, line := range lines {
+		lines[i] = "    " + line
+	}
+	return strings.Join(lines, "\n")
```

Comment:
Maybe this is slightly more efficient?
```suggestion
	return "    " + strings.ReplaceAll(s, "\n", "\n    ")
```